codalab/worker/docker_utils.py

```diff
@@ -152,7 +152,11 @@ def get_nvidia_devices(self, use_docker=True):
             docker.errors.ImageNotFound if the CUDA image cannot be pulled
             docker.errors.APIError if another server error occurs
         """
-        cuda_image = 'nvidia/cuda:12.2.0-devel-ubuntu22.04'
+
+        # Note: Do NOT update the NVIDIA image to use a CUDA version higher than
+        # that supported by the NLP machines. Otherwise, Slurm Batch Worker
+        # Manager will no longer function.
+        cuda_image = 'nvidia/cuda:11.5.2-base-ubuntu20.04'
```

Comment:
Is this image (`nvidia/cuda:11.0.3-base-ubi7`) smaller?

Reply:
Looks to me like the 11.5.2-base-ubuntu20.02 is smaller; it's [about 56 MB](https://hub.docker.com/layers/nvidia/cuda/11.5.2-base-ubuntu20.04/images/sha256-3cfedb9486eaa8f0f44a87474023397fd2929f06404576d978f04cb85c2afc2e?context=explore) while the one you list is j[ust above 89 MB](https://hub.docker.com/layers/nvidia/cuda/11.0.3-base-ubi7/images/sha256-3e5fa8cfb2724f3997dfb733f8154609472533bb5bcff3458f10fb656cceec16?context=explore).

The developer images, by the way, are huge. For reference, the `nvidia/cuda:11.5.2-devel-ubuntu20.04` is [about 2.8GB total](https://hub.docker.com/layers/nvidia/cuda/11.5.2-devel-ubuntu20.04/images/sha256-77058e8631d0e0f1c3a88cbc134c83a89720ed2e3c196970762d9a3e8b502977?context=explore)!